docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -1,258 +1,325 @@
+
+      .. rubric:: Resolved Issues
+
+      .. list-table::
+         :widths: 10 30 55
+         :header-rows: 1
+
+         * - Number
+           - Description
+           - Resolution
+         * - 1202
+           - Can't select .KIP as source
+           - Select 'header.xml' in .KIP directory to select .KIP
+         * - 1241
+           - geglxinfo should parse all meshes in terrain packets
+           - geglxinfo will parse all meshes in terrain packets
+         * - 1511
+           - Missing Systemrc.h causes compiler error
+           - Remove requirement for Systemrc.h
+         * - 1517
+           - geglxinfo should parse vector packets
+           - geglxinfo will parse vector packets
+         * - 1530
+           - Cancel command is complicated
+           - Simplify cancel command
+         * - 1532
+           - Supplemental search fields should be escaped for HTML
+             when viewing on an HTML page
+           - Escape supplemental search fields
+         * - 1538
+           - Code coverage builds incompatible with scons cache
+           - Make code coverage builds compatible with scons cache
+         * - 1540
+           - Gradle wrapper is obsolete
+           - Update gradle wrapper
+         * - 1552
+           - XML objects are not properly cleaned up
+           - A memory management bug in XML processing was fixed,
+             resulting in lower memory utilization and significantly improved
+             performance.
+         * - 1561
+           - Unused REST end point in asset manager
+           - Remove unused asset manager REST endpoint
+         * - 1564
+           - Apache does not build with crypto enabled
+           - Apache builds with crypto enabled
+         * - 1572
+           - Unused directory
+           - Remove obsolete directory
+         * - 1574
+           - Broken GLFS pointer for PyYAML tarball
+           - Fix GLFS pointer for PyYAML tarball
+         * - 1577
+           - Use defusedxml in server code
+           - Added defusedxml as a dependecy.  Using defusedxml in place of
+             PYTHON XML to improve security.
+         * - 1582
+           - Unit tests for asset version state calculation inaccurate
+           - Fix asset version state calculation unit tests.
+         * - 1586
+           - Wrong permissions on download_tutorial.sh
+           - Fix permissions issue on download_tutorial.sh
+         * - 1591
+           - New library defusedxml
+           - License file for defusedxml added.
+         * - 1593
+           - Python dependency issues for Python 2.7
+           - Resolve Python dependency issues.
+         * - 1604
+           - WMS capability doesn't work with reverse proxies
+           - It now works, as long as the protocol (http vs https) matches
+
+      .. rubric:: Known Issues
+
+      .. list-table::
+         :widths: 10 30 55
+         :header-rows: 1
+
+         * - Number
```

Comment:
Idle musing: I wonder if we could get away with removing the "Known Issues" section and simply point people to the list of issues on GitHub.

Reply:
We should see about that in the future...